feat: add `from_service_account_info`

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/7b91e2c4-9b0c-4685-ad6f-3280b28a36ba/targets

- [ ] To automatically regenerate this PR, check this box. (May take up to 24 hours.)

PiperOrigin-RevId: 364411656
Source-Link: https://github.com/googleapis/googleapis/commit/149a3a84c29c9b8189576c7442ccb6dcf6a8f95b
PiperOrigin-RevId: 361662015
Source-Link: https://github.com/googleapis/googleapis/commit/28a591963253d52ce3a25a918cafbdd9928de8cf
PiperOrigin-RevId: 361597303
Source-Link: https://github.com/googleapis/googleapis/commit/d060a12e9731b05fccb56aee4d4bf176892b7911
PiperOrigin-RevId: 359562873
Source-Link: https://github.com/googleapis/googleapis/commit/07932bb995e7dc91b43620ea8402c6668c7d102c
PiperOrigin-RevId: 355923884
Source-Link: https://github.com/googleapis/googleapis/commit/5e3dacee19405529b841b53797df799c2383536c
PiperOrigin-RevId: 354996675
Source-Link: https://github.com/googleapis/googleapis/commit/20712b8fe95001b312f62c6c5f33e3e3ec92cfaf
PiperOrigin-RevId: 352816749
Source-Link: https://github.com/googleapis/googleapis/commit/ceaaf31b3d13badab7cf9d3b570f5639db5593d9
PiperOrigin-RevId: 350246057
Source-Link: https://github.com/googleapis/googleapis/commit/520682435235d9c503983a360a2090025aa47cd1

Shabirmean commented:
The `Kokoro docs-presubmit` failure seems to be originating from the sphinx library. The repo for the library has a comment (https://github.com/sphinx-doc/sphinx/issues/5870#issuecomment-468285753) saying this was fixed in v1.7.6 of the library. 

It seems  like the sphinx version used by this repo is an older version 1.5.5 (https://github.com/googleapis/python-container/blob/master/noxfile.py#L204)

@busunkim96 has filed [this issue](https://github.com/googleapis/doc-templates/issues/94) on behalf of this in the [docs-templates repo](https://github.com/googleapis/doc-templates). It's not top priority but is in the pipeline to be fixed eventually. Until then the `docs-presubmit` checks on Kokoro would fail

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.39.1

feat: add 'from_service_account_info' factory to clients
fix: fix sphinx identifiers
PiperOrigin-RevId: 350246057

Source-Author: Google APIs <noreply@google.com>
Source-Date: Tue Jan 5 16:44:11 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 520682435235d9c503983a360a2090025aa47cd1
Source-Link: https://github.com/googleapis/googleapis/commit/520682435235d9c503983a360a2090025aa47cd1

yoshi-automation committed:
chore: update Go generator, rules_go, and protobuf

PiperOrigin-RevId: 352816749

Source-Author: Google APIs <noreply@google.com>
Source-Date: Wed Jan 20 10:06:23 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: ceaaf31b3d13badab7cf9d3b570f5639db5593d9
Source-Link: https://github.com/googleapis/googleapis/commit/ceaaf31b3d13badab7cf9d3b570f5639db5593d9

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.40.5

PiperOrigin-RevId: 354996675

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Feb 1 12:11:49 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 20712b8fe95001b312f62c6c5f33e3e3ec92cfaf
Source-Link: https://github.com/googleapis/googleapis/commit/20712b8fe95001b312f62c6c5f33e3e3ec92cfaf

yoshi-automation committed:
chore: update gapic-generator-python

PiperOrigin-RevId: 355923884

Source-Author: Google APIs <noreply@google.com>
Source-Date: Fri Feb 5 14:04:52 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 5e3dacee19405529b841b53797df799c2383536c
Source-Link: https://github.com/googleapis/googleapis/commit/5e3dacee19405529b841b53797df799c2383536c

yoshi-automation committed:
chore: update gapic-generator-python to 0.40.11

PiperOrigin-RevId: 359562873

Source-Author: Google APIs <noreply@google.com>
Source-Date: Thu Feb 25 10:52:32 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 07932bb995e7dc91b43620ea8402c6668c7d102c
Source-Link: https://github.com/googleapis/googleapis/commit/07932bb995e7dc91b43620ea8402c6668c7d102c

yoshi-automation committed:
docs: Fixed broken links in the reference documentation

PiperOrigin-RevId: 361597303

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Mar 8 10:18:21 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: d060a12e9731b05fccb56aee4d4bf176892b7911
Source-Link: https://github.com/googleapis/googleapis/commit/d060a12e9731b05fccb56aee4d4bf176892b7911

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.42.2

PiperOrigin-RevId: 361662015

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Mar 8 14:47:18 2021 -0800
Source-Repo: googleapis/googleapis
Source-Sha: 28a591963253d52ce3a25a918cafbdd9928de8cf
Source-Link: https://github.com/googleapis/googleapis/commit/28a591963253d52ce3a25a918cafbdd9928de8cf

yoshi-automation committed:
chore: upgrade gapic-generator-python to 0.43.1

PiperOrigin-RevId: 364411656

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Mar 22 14:40:22 2021 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 149a3a84c29c9b8189576c7442ccb6dcf6a8f95b
Source-Link: https://github.com/googleapis/googleapis/commit/149a3a84c29c9b8189576c7442ccb6dcf6a8f95b
